### PR TITLE
Jupyter notebook

### PR DESCRIPTION
**Pull request by timHau**
Adds a basic jupyter notebook which implements

- Canny algorithm
- a combination of canny and gaussian blur
- Harris corner Detection
- Scharr algorithm
- discrete fourier transform

Here are some example images 

701a2d77-3ac5-46da-822d-1411a2cf3939
707b0079-2df3-4121-8ce3-73d1e55e45de

### CODE REVIEW

File: code/edge_detection.ipynb [hunk 1]
```diff
@@ -0,0 +1,464 @@
+{
```
Comment (markusait) [inline code/edge_detection.ipynb:1]: <p>Great explanation!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (markusait) [inline code/edge_detection.ipynb:1]: <p>At one point we should figure out a good metric to judge the "goodness" of an edge detection. For now our best judgement should be fine, but ideally the cirrhosis detection in the last step should tell us more about it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (markusait) [inline code/edge_detection.ipynb:1]: <p>Can you elaborate a bit more on the signficance of blur_n and why it needs to be odd?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (markusait) [inline code/edge_detection.ipynb:1]: <p>Would using the gaussian blur beforehand also help here to get rid of the noise inside the image?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (markusait) [inline code/edge_detection.ipynb:1]: <p>I'd say let's iterate on the previously used algorithms and get a better understanding and results out of them before we get deeper into Fourier transformations. But great initiative!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Review by markusait — COMMENTED** [review]
Left review via ReviewNB: https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/

**Comment by markusait** [comment]
@ralph-melcher , @timHau is now helping us out now. He has submitted some great stuff via the Jupyter Notebook which we will use for ease of development now. 
The .ipynb files are unfortunately not reviewable on Github at this point, which is why I have installed Review NB and left comments there. It will allow us to elaborate on code changes much better than viewing the JSON output. 

Also, we need to make sure to reload the notebook properly with `Shit` + `Enter` from top to bottom to let the changes propagate correctly.

Going forward I think we should figure out 1. How and which edge detection algorithms to use and 2. How we want to compare the chain codes using YardStick or BoxCounting methods.

File: code/edge_detection.ipynb (hunk 1 above)
Comment (timHau) [reply code/edge_detection.ipynb:1]: blur_n is the size of the kernel used for gaussian blur ( [wikipedia](https://en.wikipedia.org/wiki/Kernel_(image_processing) ).  The current pixel always responds to the center of the kernel, which is only unique if the kernel has odd dimensions.

another way to think about it is, there will be an error if blur_n is even.

Comment (timHau) [reply code/edge_detection.ipynb:1]: definitely, we should also make sure to test on a bunch of images. Right now our solution might only work on this specific image.

**Comment by markusait** [comment]
Going to merge this in now as we need to Jupyter Notebook setup

**Pull request merged**